bindings/python/pymongoarrow/types.py

```diff
@@ -27,14 +27,16 @@ class _BsonArrowTypes(enum.Enum):
     int32 = 3
     int64 = 4
     objectid = 5
+    string = 6
 
 
 _TYPE_NORMALIZER_FACTORY = {
     Int64: lambda _: int64(),
     float: lambda _: float64(),
     int: lambda _: int64(),
     datetime: lambda _: timestamp('ms'),     # TODO: add tzinfo support
-    ObjectId: lambda _: ObjectId
+    ObjectId: lambda _: ObjectId,
+    str: lambda: string()
```

Comment:
Suggest adding a trailing comma to make future diffs nicer :)

Reply:
Done